module/utils.mjs

```diff
@@ -278,3 +279,28 @@ function _synchronizeActorSpells(actor, spellsMap) {
   }
   return {toDelete, toCreate};
 }
+
+/**
+ * For unidentified items, returns a localized masked name based on the item type and base item, where appropriate.
+ * @param {object} item   The item to evaluate.
+ * @returns {string}      The localized name to show for the unidentified item.
+ * @private
+ */
+ export function unidentifiedName(item) {
```

Comment:
None of these are properly localized:

<img width="564" alt="Screen Shot 2022-10-13 at 11 37 07" src="https://user-images.githubusercontent.com/19979839/195679241-d17754a0-a953-4580-add4-327797d853c7.png">

<img width="559" alt="Screen Shot 2022-10-13 at 11 40 02" src="https://user-images.githubusercontent.com/19979839/195679475-159602eb-d958-4d33-96af-2833cb777f7c.png">

Reply:
Oops, I thought they were in the system! Thanks for the review, I will look into it asap (I am moving this weekend though so it might take some time to catch up)